.travis.yml

```diff
@@ -15,19 +15,24 @@ addons:
       # required for php7.3
       - libzip4
 
-script:
-  - ./bin/phpunit --configuration tests/phpunit.xml --coverage-clover=coverage.xml
-  - ./bin/sabre-cs-fixer fix . --dry-run --diff
+install:
+  - mkdir -p tools
+  - wget -nc -P tools https://cs.sensiolabs.org/download/php-cs-fixer-v2.phar
```

Comment:
I would prefer prettyci instead of a travis build formthe style check
https://prettyci.com/

That way the travis builds dont last even longer

Reply:
cool - I'll have a look. I was not aware of this service. THX

Reply:
I just added prettyci to all sabre repositories.

We only need the nen .trvais.yml changes for this PR then

Reply:
Reverted all this ....